docs/README.md

```diff
@@ -277,9 +277,10 @@ saltgui_ipnumber_prefix:
 ```
 The display of the IP-numbers can simply be disabled by choosing a non-existing grain or by choosing a non-existing prefix.
 
+### Cached Grains
```

Comment:
the rest of this chapter is not divided in 3rd-level sections. it either should be left out, or a section "### Regular Grains" should be added just below "## Grains"